Packs/NucleonCyber/Integrations/NucleonCyberFeed/NucleonCyberFeed.py

```diff
@@ -0,0 +1,761 @@
+        }
+        indicators.append(indicator_obj)
+
+    return indicators
+
+
+def get_indicators_command(client: Client,
+                           params: Dict[str, str],
+                           args: Dict[str, str]
+                           ) -> CommandResults:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+        args: demisto.args()
+    Returns:
+        Outputs.
+        IP indicators
+    """
+    limit = int(args.get('limit', '10'))
+    tlp_color = params.get('tlp_color')
+    feed_tags = argToList(params.get('feedTags', ''))
+    indicators = fetch_indicators(client, tlp_color, feed_tags, limit)
+    human_readable = tableToMarkdown('IP indicators from NucleonCyberFeed:', indicators,
+                                     headers=['value', 'type', 'exp'], headerTransform=string_to_table_header, removeNull=True)
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='NucleonCyber.Indicators',
+        outputs_key_field='',
+        raw_response=indicators,
+        # outputs={},
+        outputs=indicators,
+    )
+
+
+def fetch_indicators_command(client: Client, params: Dict[str, str]) -> List:
+    """Wrapper for fetching indicators from the feed to the Indicators tab.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+    Returns:
+        All indicators type (ip,url,hash(file)).
+    """
+    feed_tags = argToList(params.get('feedTags', ''))
+    tlp_color = params.get('tlp_color')
+    ips = fetch_indicators(client, tlp_color, feed_tags)
+    urls = fetch_urls(client)
+    hashes = fetch_hashes(client)
+    return [ips, urls, hashes]
+
+
+"""
+"""
+
+
+def get_hashes_command(client: Client,
+                       params: Dict[str, str],
+                       args: Dict[str, Any]
+                       ) -> CommandResults:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+        args: demisto.args()
+    Returns:
+        Outputs.
+        Hash indicators
+    """
+    limit = int(args.get('limit', '10'))
+    hashes = fetch_hashes(client, limit)
+    human_readable = tableToMarkdown('Hash indicators from NucleonCyberFeed:', hashes,
+                                     headers=['value', 'type'], headerTransform=string_to_table_header, removeNull=True)
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='NucleonCyber.Indicators.hash',
+        outputs_key_field='',  # in Xsoar it helps to update exist instead of adding new
+        raw_response=hashes,
+        outputs=hashes,
+    )
+
+
+"""
+"""
+
+
+def get_urls_command(client: Client,
+                     params: Dict[str, str],
+                     args: Dict[str, Any]
+                     ) -> CommandResults:
```

Comment:
Please remove params as it is not used in function
```suggestion
def get_urls_command(client: Client,
                     args: Dict[str, Any]
                     ) -> CommandResults:
```

Reply:
removed